bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tgraph.py

```diff
@@ -34,9 +36,30 @@ def set_size(self, buf):
 # The pythonization consists in setting the size of the array that the getter
 # method returns, so that it is known in Python and the array is fully usable
 # (its length can be obtained, it is iterable).
-comp = cppyy.py.compose_method('^TGraph(2D)?$|^TGraph.*Errors$', # class to match
-                               'GetE?[XYZ]$',                    # method to match
-                               set_size)                         # post-process function
+comp = cppyy.py.compose_method(
```

Comment:
Could you leave this piece of code with its original indentation? The comments next to the arguments help understand what they do.